packages/corvid-e2e-tests/test/drivers/cliDriver.js

```diff
@@ -1,29 +1,63 @@
 const execa = require("execa");
 const which = require("npm-which")(__dirname);
-
+const { findFreePort } = require("./utils");
 const CORVID_BIN_PATH = which.sync("corvid");
 
 module.exports = ({ cwd }) => {
   const parseCommandArgs = ({ remoteDebuggingPort } = {}) =>
     remoteDebuggingPort ? `--remote-debugging-port=${remoteDebuggingPort}` : "";
 
-  const login = ({ remoteDebuggingPort } = {}) => {
-    const query = parseCommandArgs({ remoteDebuggingPort });
-    return execa.shell(`${CORVID_BIN_PATH} login ${query}`, { cwd });
+  const login = async ({ remoteDebuggingPort } = {}) => {
```

Comment:
do we still need the option to pass the port as an argument ?

Reply:
we don't really need it, but we said that we will keep the option to pass a port to the cli command..

Reply:
actually i think each command should just get a string of arguments in the same way a user would use the actual cli.
in that case, we will always be able to just add "--remote-debugging-port=12345".
and we will also not have to update the cli driver when we allow new options for existing commands

Reply:
in second thought, in the current implementation the cli driver is adding the remoteDebuggingPort statment for the users and returing the port, if we will allow to pass the extraStringArgs then we will need to analyse the string argument to extract the actual port and for avoiding the situation when the command will have 2 remoteDebuggingPort statements..
“--remote-debugging-port=OUR_RANDOM_PORT --remote-debugging-port=USER_CUSTOM_PORT”
So I am not sure if it’s the correct path to go..